build-system/tasks/helpers.js

```diff
@@ -457,6 +443,69 @@ function compileJs(srcDir, srcFilename, destDir, options) {
   }
 }
 
+/**
+ * Enables local testing mode for various target files
+ * @param {string} destFilename
+ */
+function maybeEnableLocalTesting(destFilename) {
+  if (process.env.NODE_ENV === 'development') {
+    if (destFilename === 'amp.js') {
+      return enableLocalTesting('dist/amp.js');
```

Comment:
should we create a map for this and just loop over it?

Reply:
Yes, that's the intention. Coming up in a new PR. I'm just moving code here.

Reply:
Sent out for review in #22256